fineract-provider/src/main/java/org/apache/fineract/cob/loan/ApplyChargeToOverdueLoansBusinessStep.java

```diff
@@ -35,13 +37,18 @@ public class ApplyChargeToOverdueLoansBusinessStep implements LoanCOBBusinessSte
 
     @Override
     public Loan execute(Loan loan) {
+        setupActionContext();
```

Comment:
I think we shouldn't set up the ActionContext here but rather in the COB itself before its starting to run all the business step, what do you think?

Reply:
Yeah, it makes sense to setup globally as well. For reference i am keeping this to show if someone wanna use different actionContext, this is how and when it should be configured.